### PR TITLE
Specify a device type

### PR DESCRIPTION
**Pull request by eroldan**
In reference to issue #50

### CODE REVIEW

**Comment by McGiverGim** [comment]
I don't know if maybe it's better to add simply an "extra parameters" that can work with any parameter/s, and use the device type as example in the documentation.
Has smartctl other parameters that can be useful?

**Comment by eroldan** [comment]
Having a generic field is something that I thought about but could not find any other useful parameter in my use cases. I think that `hdd_path` and `device_type` will cover 99% of usage. Still is doable to add a generic field later if there is need from users. It's your call. Thank for reviewing this.

File: hdd_tools/config.json
Comment (McGiverGim) [inline hdd_tools/config.json:28]: Move this too after `hdd_path`.

File: hdd_tools/config.json
Comment (McGiverGim) [inline hdd_tools/config.json:43]: Move this too after `hdd_path`.

The easier way for users is to make this a list with all the possible options, but I have seen different documentation, some of them explain 10 or 20 possible values, that is ok, but others comment more than 200 because you can select the device number... do you know how many different options exist?

File: hdd_tools/translations/en.yaml
Comment (McGiverGim) [inline hdd_tools/translations/en.yaml]: Move this too after `hdd_path`.

All the descriptions in the file start with uppercase and one little typo (explicityly -> explicitly) :

But maybe is better to simply copy&paste the same text that you added to the official documentation, it seems ok to me:
> Type of block device which `smartctl` will try to use for communication. Default "auto". Check https://www.smartmontools.org/wiki/USB

File: hdd_tools/DOCS.md
Comment (McGiverGim) [inline hdd_tools/DOCS.md:21]: I think we need to change the parameter position. We have the "options" for the command up, then performance, debug, output, etc. so better put it for example after `hdd_path`, it seems more natural to me. What do you think?

**Review by McGiverGim — CHANGES_REQUESTED** [review]
Sorry for my late response... it seems ok to me, let's merge it. But before that, only minor changes...

**Comment by hubikj** [comment]
Hello, any chance this will be merged? I suffer from the same issue, while `smartctl --all -d sntjmicron /dev/sda` works for me. Thanks!

**Comment by McGiverGim** [comment]
@eroldan are you interested about finishing this?

File: hdd_tools/config.json
Comment (eroldan) [reply hdd_tools/config.json:43]: I don't think that maintaining all the possible options is reasonable. Just let the user read the documentation and place a value.

Comment (McGiverGim) [reply hdd_tools/config.json:43]: Ok. Seems reasonable to me.

**Comment by eroldan** [comment]
Hi @McGiverGim ,

I think the current state works for me and probably other users. I don't have much time now. I recommend you to just merge and do modifications to your preferences. I still use this addon. Thanks!

**Comment by McGiverGim** [comment]
Ok, I can't merge it directly because the conflict, it needs a rebase, but I will modify it to merge.

**Pull request closed without merging**
